go-fuzz-dep/main_libFuzzer.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2015 go-fuzz project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+// +build gofuzz
+// +build gofuzz_libfuzzer
+
+package gofuzzdep
+
+import (
+	"unsafe"
+
+	. "github.com/dvyukov/go-fuzz/go-fuzz-defs"
+)
+
+// Bool is just a bool.
```

Comment:
I'd like to factor the shared code into a third file that is only protected by the `gofuzz` build tag.

But again, I'm happy to do that myself as a follow-up after this is merged.